[v1.60.0] Triggered Release Notes (#655)
replicatedhq/kots.io: content/release-notes/1.60.0.md

---
date: 2022-01-25T10:21:28-07:00
linktitle: "1.60.0"
title: "1.60.0"
kubernetes: "1.20, 1.21, and 1.22"
---

{{<features>}}

> * New user inteface on the Dashboard and Version history pages.

Reviewer: ```suggestion
* New user interface on the Dashboard and Version history pages.
```

> * New user inteface on the Dashboard and Version history pages.

Reviewer: @ajp-io I did an initial quick pass on this, but let me know when you're done so I can re-review/approve. For this new feature I feel like we're still a bit vague here. I wasn't sure what PM wanted to highlight here, but tossed together the below suggestion from the blog post on it as a jumping off point.

```suggestion
* The admin console app dashboard and version history pages are redesigned so that you can view and sync your license, see the configured snapshot storage destination, take a snapshot, configure automatic updates, and more right from the dashboard.
```

Reviewer 2: Thanks a lot. I pushed a change with some wording we could use. I don't know your opinion on exclamation points in release notes, but I put one in there and linked to the blog with more details.

Reviewer: I'm good with that :) Felt like it deserved more love in the RN and we accomplished that. Thanks!

{{</features>}}

{{<changes>}}
* Updates Minio to RELEASE.2022-01-08T03-11-54Z (resolves CVE-2021-43858 CVE)

> * Updates Postgres to 10.19

Reviewer: ```suggestion
* Updates Postgres to version 10.19
```

{{</changes>}}

{{<fixes>}}

> * Fixes an issue that caused images to be pushed multiple times during [airgap installation](/kotsadm/installing/airgap-packages/#upload-airgap-bundle) when [Native Helm](/vendor/helm/helm-processing/#native-helm) feature is enabled.

Reviewer: ```suggestion
* Fixes an issue that caused images to be pushed multiple times during an [airgap installation](/kotsadm/installing/airgap-packages/#upload-airgap-bundle) when the [Native Helm](/vendor/helm/helm-processing/#native-helm) feature is enabled.
```

> * Prevent deployment status labels from breaking into multiple lines on small displays

Reviewer: ```suggestion
* Fixes an issue that prevented the deployment status labels from breaking into multiple lines on small displays.
```
